sdk/src/androidTest/java/com/desk/android/sdk/util/ViewMatchers.java

```diff
@@ -1,41 +1,21 @@
-/*
```

Comment:
Why was the license header removed? This needs to be here. Also, can you add the header to any other classes in the `androidTest` source code? I noticed some where missing (not your fault, but they need to get added).